src/poetry/utils/exporter.py

```diff
@@ -120,11 +120,10 @@ def _export_requirements_txt(
                     hashes.append(f"{algorithm}:{h}")
 
                 if hashes:
-                    line += " \\\n"
+                    sep = " \\\n"
```

Comment:
This can be a raw string instead of using so many escapes.

Reply:
this is a literal `\` char + a newline `\n`. couldn't figure out a more concise way to state that even with raw strings (note that `print(r"\" + "\n")` is invalid)